src/Control/Monad/Trans/Control.hs

```diff
@@ -499,6 +503,17 @@ instance Monoid w => MonadTransControl (Strict.RWST r w s) where
     {-# INLINABLE liftWith #-}
     {-# INLINABLE restoreT #-}
 
+#if MIN_VERSION_transformers(0,5,3)
```

Comment:
We should depend on recent enough `transformers-compat` to avoid needing CPP, if only possible.

Reply:
How does [this](https://github.com/basvandijk/monad-control/commit/f83314c8e700a21839f1215a2f8637fd778a7eda) look?